profile/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
Don't include .idea files (most dotfiles shouldn't be in github). You can use git rm to remove them, and use your .gitignore to prevent future dotfiles from being added.